src/main/java/ca/corefacility/bioinformatics/irida/util/IridaFiles.java

```diff
@@ -65,4 +66,34 @@ public static String getFileExtension(List<? extends SequencingObject> files) th
 		return iridaFileStorageUtility.getFileExtension(files);
 	}
 
+	/**
+	 * Cleans up temporary downloaded files.
+	 *
+	 * @param filePath The path to the file
+	 * @param directoryPath The path to the directory which has the file
+	 */
+	public static void cleanupDownloadedLocalTemporaryFiles(Path filePath, Path directoryPath) {
+		iridaFileStorageUtility.cleanupDownloadedLocalTemporaryFiles(filePath, directoryPath);
+	}
+
+	/**
+	 * Cleans up temporary files.
+	 *
+	 * @param filePath The path to the file
+	 * @param directoryPath The path to the directory which has the file
+	 */
+	public static void cleanupLocalTemporaryFiles(Path filePath, Path directoryPath) {
```

Comment:
If it makes sense can we just leave this method accessable form the `IridaFileStorageUtility` class and not in `IridaFiles`?  That way it's only responsible for dealing with those temporarily downloaded files.

Reply:
Removed this method in 1dbdb47af3eedeee57410916380b562e92a85c41 as it is not required. We have the `cleanupDownloadedTemporaryFiles` method in the irida files storage utility classes to clean up any files downloaded from an object store